lib/diff.mli

```diff
@@ -35,54 +39,50 @@ and type_modification = {
 }
 
 and type_kind =
-  | Record_tk of (Types.label_declaration, record_field) map
-  | Variant_tk of
-      ( Types.constructor_declaration,
-        (Types.constructor_declaration, cstr_args) atomic_variant )
-      map
-
-and record_field = {
-  type_expr : Types.type_expr atomic_maybe_changed;
-  mutable_flag : (Asttypes.mutable_flag, field_mutability) maybe_changed;
+  | Record_tk of (Types.label_declaration, label) map
+  | Variant_tk of (Types.constructor_declaration, cstr_args) map
```

Comment:
Why not use the `variant` label here, instead of having a `Atomic_cstr` in `cstr_args`?